Contributing.md

```diff
@@ -93,7 +93,7 @@ git push origin YourBranchName
 
 ## 💥 Issues
 
-For major changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
+For large/discussable changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```

Comment:
I think major sounds better here

Reply:
Again, large/discussable really doesn’t sound right here, I’m not sure discussable is even a real word

Reply:
```suggestion
For major changes and controversial/opinionated changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```

Reply:
I think we can probably mention -> 
```
For discussing changes, you are welcome to [open an issue](https://github.com/EddieHubCommunity/LinkFree/issues/new/choose) about what you would like to contribute. Enhancements are always encouraged and appreciated.
```

Reply:
Because regardless, we encourage people to open an issue first.